capstone/src/main/java/com/google/sps/servlets/FeedbackServlet.java

```diff
@@ -34,27 +34,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String problemArea = getParameter(request, "problem-area-input", "");
     String suggestion = getParameter(request, "suggestion-input", "");
     String otherFeedback = getParameter(request, "feedback-input", "");
-
+ 
     Entity feedbackEntity = new Entity("Feedback");
     feedbackEntity.setProperty("name", name);
     feedbackEntity.setProperty("relation", relation);
     feedbackEntity.setProperty("problemArea", problemArea);
     feedbackEntity.setProperty("suggestion", suggestion);
     feedbackEntity.setProperty("otherFeedback", otherFeedback);
-
+ 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(feedbackEntity);
-
+ 
     response.sendRedirect("/main.html");
   }
-
-
- /**
+ 
+  /*
    * @return the request parameter, or the default value if the parameter was not specified by the
    *     client
    */
   private String getParameter(HttpServletRequest request, String paramName, String defaultValue) {
     String value = request.getParameter(paramName);
-      return value || defaultValue;
+      if (value.length() == 0) {
```

Comment:
might be more clear if you did `value.isEmpy()``

Reply:
tru